zerver/lib/actions.py

```diff
@@ -2010,6 +2011,20 @@ def do_change_left_side_userlist(user_profile, setting_value, log=True):
         log_event(event)
     send_event(event, [user_profile.id])
 
+def mark_user_star_backend(user_profile, email):
+    starred_user = get_object_or_404(UserProfile, email=email)
```

Comment:
I've not seen this before--does it work as expected? I'd expect `json_error` here.